### ISSUE TITLE
containerized build - etcd not starting.

### ISSUE THREAD

**Issue by ianmiell** [IshentRas/cookbook-openshift3#276]
I can repro this, but the systemd process fails on:

```
[root@master1 system]# /usr/bin/docker run --name etcd_container --rm -v /var/lib/etcd:/var/lib/etcd:z -v /etc/etcd:/etc/etcd:ro --env-file=/etc/etcd/etcd.conf --net=host --entrypoint=/usr/bin/etcd quay.io/coreos/etcd
container_linux.go:247: starting container process caused "exec: \"/usr/bin/etcd\": stat /usr/bin/etcd: no such file or directory"
/usr/bin/docker-current: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "exec: \"/usr/bin/etcd\": stat /usr/bin/etcd: no such file or directory".
```

**Comment by ianmiell** [comment]
It should be /usr/local/bin/etcd

**Comment by IshentRas** [comment]
Should be fixed now. All good. Tested all versions (rpm and containerised)
It also helped to fixed another opened bug around etcd 3.10 static pod :+1:
